setup.py

```diff
@@ -0,0 +1,59 @@
+import re
+from gzip import READ
+from pathlib import Path
+from urllib import request
+
+from setuptools import find_packages, setup
+
+README = request.urlopen('https://raw.githubusercontent.com/ultralytics/yolov5/master/README.md').read().decode('utf-8')
```

Comment:
Hi @AyushExel , Is it possible to read the README.md file directly here, as it would be difficult to use in some areas if reading it from the network, and it would also cause difficulties when compiling in an environment without network.

Reply:
Yes @zhiqwang great point. I'll update this to read the readme.md file from the local dir. Thanks for pointing it out!
Also feel free to test this and provide any other feedbacks/suggestions that you might have.

Reply:
Yea this is a good point. The previous example I used network read because we were using a remote README, but that doesn’t apply here